### PR TITLE
Improve how-to-write-and-deploy-an-nft tutorial

### PR DESCRIPTION
**Pull request by neozapatista**
Changes:

* add missing await which can cause deployment to fail
* move ` --network ropsten` argument to avoid confusion with scripts/deploy.js

### CODE REVIEW

File: src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md [hunk 1]
```diff
@@ -319,7 +320,7 @@ Calling deploy() on a ContractFactory will start the deployment, and return a Pr
 
 We’re finally ready to deploy our smart contract! Navigate back to the root of your project directory, and in the command line run:
 
-    npx hardhat run scripts/deploy.js --network ropsten
+    npx hardhat run scripts/deploy.js
```
Comment (corwintines) [inline src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:323]: I think we should be keeping a test network here for the deploy to protect user funds and avoid confusion.

Comment (corwintines) [reply src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:323]: If you think there is a better testnet though, please do change

Comment (neozapatista) [reply src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:323]: I thought this was being ignored, as the contents of scripts/deploy.js in the same file don't look at command line arguments. I realize now this flag is meant for `hardhat`, so I'll add it back and place it before `run` to make that clear.

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for spotting and fixing this @neozapatista ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @neozapatista for docs

**Comment by Aurel57290** [comment]
Hello !

First Thank you for this well explained tutorial.
During the installation of libraries I face an issue, more especially with the following script : 
**npm install --save-dev @nomiclabs/hardhat-ethers 'ethers@^5.0.0'**
and the log in the console : 
C:\Users\aurelien\Desktop\Dossier_Py\test>npm install --save-dev @nomiclabs/hard
hat-ethers 'ethers@^5.0.0'
_npm ERR! code E404_
_npm ERR! 404 Not Found - GET https://registry.npmjs.org/%27ethers - Not found_
_npm ERR! 404_
_npm ERR! 404  ''ethers@5.0.0'' is not in this registry._
_npm ERR! 404 This package name is not valid, because_
_npm ERR! 404  1. name can no longer contain special characters ("~'!()*")_
_npm ERR! 404_
_npm ERR! 404 Note that you can also install from a_
_npm ERR! 404 tarball, folder, http url, or git url._

_npm ERR! A complete log of this run can be found in:_
_npm ERR!     C:\Users\aurelien\AppData\Local\npm-cache\_logs\2022-01-13T17_08_38_

Would you have an idea about the problem ? If I remove quote in 'ethers@^5.0.0', would it be correctly installed ?

Thank you very much for your help !
Aurélien

**Comment by neozapatista** [comment]
Thanks @minimalsm, would love to join the discord. I've tried signing up but they require a phone number and I'm not comfortable giving them one. Don't mean to complain, just let you know why I'm not joining. Invite appreciated nonetheless.

Screenshot 2022-01-16 at 21 01 17

Screenshot 2022-01-16 at 21 01 20

**Pull request merged**